gunicorn/util.py

```diff
@@ -53,6 +56,14 @@ def _setproctitle(title):
     def _setproctitle(title):
         pass
 
+def load_entry_point(distribution, group, name):
```

Comment:
`gunicorn\util.py:59:1: E302 expected 2 blank lines, found 1`
```suggestion

def load_entry_point(distribution, group, name):
```